projects/views.py

```diff
@@ -8,11 +8,13 @@
 
 # Create your views here.
 # done test view
-def done_view(request):
-    data = Feature.objects.all()
+def done_view(request, project_id):
+    data_p = Project.objects.get(id=project_id)
+    data_f = Feature.objects.all()
```

Comment:
Not necessary to query the database for all Features, the associated Features in the ManyToOne relationship are already on the `data_p` Project object.